plugins/woocommerce/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -90,7 +90,34 @@ protected function initialize() {
 		parent::initialize();
 		add_action( 'wp_loaded', array( $this, 'register_empty_cart_message_block_pattern' ) );
 		add_action( 'wp_print_footer_scripts', array( $this, 'print_lazy_load_scripts' ), 2 );
-		add_filter( 'hooked_block_types', array( $this, 'register_hooked_block' ), 10, 4 );
+		/**
+		 * The hooked_block_{$hooked_block_type} filter was added in WordPress 6.5.
+		 * We are the only code adding the filter 'hooked_block_woocommerce/mini-cart'.
+		 * Using has_filter() for a compatibility check won't work because add_filter() is used in the same file.
+		 */
+		if ( version_compare( get_bloginfo( 'version' ), '6.5', '>=' ) ) {
```

Comment:
I would remove this check given that WooCommerce support is L-1.

Reply:
done -> [fc6e384](https://github.com/woocommerce/woocommerce/pull/53583/commits/fc6e384ce55c53901b6e7154a7844c3083ac9afb)